pkg/cluster/controllers/storage/provision.go

```diff
@@ -88,7 +89,7 @@ func (m *MinIOController) getMinIOProperties(minioInstance *minio.Tenant) (*goha
 		endpoint = fmt.Sprintf("http://%s.%s.svc:%s", m.getServiceName(), m.HarborCluster.Namespace, "9000")
 	}
 
-	storageSpec.S3.RegionEndpoint = endpoint
+	storageSpec.S3.RegionEndpoint = strings.ToLower(endpoint)
```

Comment:
http://serverA and http://servera is the same thing?

Reply:
The DNS names are case insensitive